sympy/holonomic/holonomic.py

```diff
@@ -1036,14 +1036,24 @@ def evalf(self, points, **kwargs):
         >>> # a straight line on the real axis from (0 to 1)
         >>> r = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
 
-        >>> HolonomicFunction(Dx - 1, x, 0, [1]).evalf(r)  # e^1
+        # using Runge-Kutta 4th order on e^x from 0.1 to 1.
+        # exact solution at 1 is 2.71828182845905
+        >>> HolonomicFunction(Dx - 1, x, 0, [1]).evalf(r)
+        [1.10517083333333, 1.22140257085069, 1.34985849706254, 1.49182424008069, 
```

Comment:
I fixed it at https://github.com/sympy/sympy/pull/11205. See also https://github.com/sympy/sympy/issues/11204.